index.html

```diff
@@ -5503,8 +5503,11 @@ <h3>Algorithm</h3>
                   of <var>value</var> equals <code>xsd:boolean</code>, set
                   <var>converted value</var> to <code>true</code> if the
                   <a>lexical form</a>
-                  of <var>value</var> matches <code>true</code>, or <code>false</code>
-                  if it matches <code>false</code>. If it matches neither,
+                  <ins cite="#change_5">
+                    of <var>value</var> matches <code>"true"</code> or <code>"1"</code>, or <code>false</code>
+                    if it matches <code>"false"</code> or <code>"0"</code>.
+                  </ins>
+                  If it matches neither,
```

Comment:
I suggest to use `<ins>` in a more targeted way. The goal is that the text *without* the `<ins>` elements matches the text before the change. (an alternative being to keep the old text inside a `<del>` element, and the new one inside a `<ins>` element, but that's not required here).

```suggestion
                  
                    of <var>value</var> matches <code>"true"</code><ins cite="#change_5"> or <code>"1"</code></ins>, or <code>false</code>
                    if it matches <code>"false"</code><ins cite="#change_5"> or <code>"0"</code></ins>.
                  </ins>
                  If it matches neither,
```
```suggestion
                  <ins cite="#change_5">
                    of <var>value</var> matches <code>"true"</code> or <code>"1"</code>, or <code>false</code>
                    if it matches <code>"false"</code> or <code>"0"</code>.
                  </ins>
                  If it matches neither,
```